src/composables/multisigAccounts.ts

```diff
@@ -44,6 +45,20 @@ function getStoredMultisigAccounts(networkId: string): IMultisigAccount[] {
   return storedMultisig ? JSON.parse(storedMultisig) : [];
 }
 
+const getNonce = async (instance: any) => (await instance.methods.get_nonce()).decodedResult;
+
+const getSigners = async (instance: any) => (await instance.methods.get_signers()).decodedResult;
+
+const getVersion = async (instance: any) => (await instance.methods.get_version()).decodedResult;
+
+const getConsensus = async (instance: any) => (await instance.methods.get_consensus_info())
+  .decodedResult;
+
+const getContractInstance = async (contractId: string, sdk: ISdk) => sdk.getContractInstance({
+  aci: SimpleGAMultiSigAci,
+  contractAddress: contractId,
+});
```

Comment:
In my opinion this method gives no value - it's just a shortcut used once. Using `sdk.getContractInstance` directly is a better option I think.